USERS.md

```diff
@@ -260,3 +260,4 @@ Currently, the following organizations are **officially** using Argo CD:
 1. [Yubo](https://www.yubo.live/)
 1. [Zimpler](https://www.zimpler.com/)
 1. [ZOZO](https://corp.zozo.com/)
+1. [Omni](https://omni.se/)
```

Comment:
Thanks for the PR @karol-szymanowski! Would you mind moving this up to make it alphabetical order?

Reply:
Sure 👍